test/core/rules/docstring_test.py

```diff
@@ -35,8 +36,21 @@ def test_keyword_anti_before_best():
             if rule._check_docstring is True:
                 assert rule.__doc__.index(KEYWORD_ANTI) < rule.__doc__.index(
                     KEYWORD_BEST
-                ), f"{rule.__name__} keyword {KEYWORD_BEST} appears before "
-                f"{KEYWORD_ANTI}"
+                ), (
+                    f"{rule.__name__} keyword {KEYWORD_BEST} appears before "
+                    f"{KEYWORD_ANTI}"
+                )
+
+
+def test_config_decorator():
+    """Test docstring anti pattern before best pattern."""
+    for plugin_rules in get_plugin_manager().hook.get_rules():
+        for rule in plugin_rules:
+            if hasattr(rule, "config_keywords"):
+                assert is_configurable(rule), (
+                    f"Rule {rule.__name__} has config but is not decorated to display "
+                    "the config."
+                )
```

Comment:
An actual bit of code added in this PR! 😁

So worth reviewing this. But tested, and it works, and helped me identify all the rules missing the decorator.